tests/cpp/operator/test_normalization.cu

```diff
@@ -245,58 +279,68 @@ void performTest(const size_t N, const size_t H, const bool zero_centered_gamma)
+    rtol_bwd = 8e-3;
+  }
   compareResults("dx", dx, ref_dx.get(), atol_bwd, rtol_bwd);
   compareResults("dgamma", dgamma, ref_dgamma.get(), atol_bwd, rtol_bwd);
   compareResults("dbeta", dbeta, ref_dbeta.get(), atol_bwd, rtol_bwd);
 }
 
-std::vector<std::pair<size_t, size_t>> test_cases = {{2048, 12288},
-                                                     {768, 1024},
-                                                     {256, 65536},
-                                                     {128, 6144},
-                                                     {64, 2304},
-                                                     {229, 541},   // Primes 50, 100
-                                                     {71, 3571},   // Primes 20, 500
-                                                     {29, 17389}}; // Primes 10, 2000
+std::vector<std::pair<size_t, size_t>> test_cases = {
+  {2048, 12288},
+  {768, 1024},
+  {256, 65536},
+  {128, 6144},
+  {64, 2304},
+  {229, 541},   // Primes 50, 100
+  {71, 3571},   // Primes 20, 500
+  {29, 17389} // Primes 10, 2000
+};
 
 }  // namespace
 
-class LNTestSuite : public ::testing::TestWithParam<std::tuple<transformer_engine::DType,
+class NormTestSuite : public ::testing::TestWithParam<std::tuple<NormType,
+transformer_engine::DType,
                                                                transformer_engine::DType,
                                                                std::pair<size_t, size_t>,
                                                                bool>> {};
 
-TEST_P(LNTestSuite, TestLN) {
+TEST_P(NormTestSuite, TestNorm) {
     using namespace transformer_engine;
     using namespace test;
 
-    const DType input_type = std::get<0>(GetParam());
-    const DType output_type = std::get<1>(GetParam());
-    const auto size = std::get<2>(GetParam());
-    const bool zero_centered_gamma = std::get<3>(GetParam());
+  const NormType norm_type = std::get<0>(GetParam());
+    const DType input_type = std::get<1>(GetParam());
+    const DType output_type = std::get<2>(GetParam());
+    const auto size = std::get<3>(GetParam());
+    const bool zero_centered_gamma = std::get<4>(GetParam());
 
     TRANSFORMER_ENGINE_TYPE_SWITCH_ALL(input_type, InputType,
       TRANSFORMER_ENGINE_TYPE_SWITCH_ALL(output_type, OutputType,
-        performTest<InputType, OutputType>(size.first, size.second, zero_centered_gamma);
+        performTest<InputType, OutputType>(size.first, size.second, zero_centered_gamma, norm_type);
       );
     );
 }
 
 INSTANTIATE_TEST_SUITE_P(
     OperatorTest,
-    LNTestSuite,
+    NormTestSuite,
     ::testing::Combine(
+          ::testing::Values(NormType::LayerNorm, NormType::RMSNorm),
         ::testing::Values(DType::kFloat32, DType::kBFloat16, DType::kFloat16),
         ::testing::Values(DType::kFloat32, DType::kBFloat16, DType::kFloat16, DType::kFloat8E4M3),
         ::testing::ValuesIn(test_cases),
         ::testing::Values(false, true)),
```

Comment:
It would be helpful to test both cuDNN and TE kernels. We can reduce the number of test shapes to keep the total number of tests manageable.